### PR TITLE
[SYCL][Matrix] Fix the bug in element_wise_all_ops_tf32_impl.hpp

### PR DESCRIPTION
**Pull request by arnamoy10**

### CODE REVIEW

**Review by yubingex007-a11y — APPROVED** [review]

**Comment by arnamoy10** [comment]
ping @intel/llvm-reviewers-runtime @intel/llvm-gatekeepers

File: sycl/test-e2e/Matrix/element_wise_all_ops_tf32_impl.hpp [hunk 1]
```diff
@@ -52,7 +52,7 @@ void matrix_verify_add(queue q, big_matrix<Ts, M, N> &A, nd_range<2> &r,
            ext::intel::experimental::matrix::joint_matrix_store(
                sg, sub_a,
                accA.template get_multi_ptr<access::decorated::no>() +
-                   (sg_startx * TM) * N + sg_starty / SG_SZ * TN,
+                   (sg_startx * TM) * N + sg_starty / SG_SZ * TK,
```
Comment (YuriPlyakhin) [inline sycl/test-e2e/Matrix/element_wise_all_ops_tf32_impl.hpp:55]: should it be `(sg_startx * TM) * K + sg_starty / SG_SZ * TK` ?

File: sycl/test-e2e/Matrix/element_wise_all_ops_tf32_impl.hpp [hunk 2]
```diff
@@ -224,7 +224,7 @@ int main() {
   big_matrix<float, MATRIX_M, MATRIX_N> MA((float *)&A);
 
   size_t NDRangeM = MATRIX_M / TM;
-  size_t NDRangeN = MATRIX_N / TN;
+  size_t NDRangeN = MATRIX_N / TK;
```
Comment (YuriPlyakhin) [inline sycl/test-e2e/Matrix/element_wise_all_ops_tf32_impl.hpp:227]: should it be `size_t NDRangeK = MATRIX_K / TK;` ?

**Review by YuriPlyakhin — COMMENTED** [review]
I think this test should be updated to use M and K consistently for matrix A (instead of using M and N in some places and M and K in other places) to avoid potential errors and confusion.

**Comment by arnamoy10** [comment]
> 
Agreed.  Here is a PR https://github.com/intel/llvm/pull/10598

**Pull request merged**